source/help/getting-started/messaging-basics.rst

```diff
@@ -0,0 +1,45 @@
+Messaging Basics
+================
+
+--------------
+
+**Write messages** using the text input box at the bottom of Mattermost.
+Press **ENTER** to send a message. Use **Shift+ENTER** to create a new
```

Comment:
We should come up with a guideline on whether to capitalize "Shift", "Alt", and other keyboard buttons in our docs. We're really inconsistent right now.

![image](https://cloud.githubusercontent.com/assets/13119842/20758735/c3e321f0-b6e8-11e6-9f17-948715ad313b.png)

I looked into this previously, and there isn't really a standard as far as I know. We however use capital letters in `shortcuts` and keyboard shortcuts docs (+ in a few other places), so propose using capitals.

Up for discussion though.